assets/components/content-types/coursebook/coursebook.scss

```diff
@@ -1 +1,192 @@
 @charset 'utf-8';
+
+/**
+* Program
+**/
+.coursebook-program {
+
+  .tree {
+    padding: 0;
+    list-style-type: none;
+  }
+
+  .tree > li {
+    flex: 0 0 50%;
+    font-size: $font-size-sm;
+
+    strong {
+      font-size: $font-size-sm * 0.9;
+    }
+
+    .underline {
+      display: inline;
+    }
+
+    .underline > a {
+      text-decoration: none;
+    }
+
+    > ul {
+      padding: 0;
+      list-style-type: none;
+      font-size: $font-size-sm;
+
+      li {
+        display: flex;
+        flex-flow: row wrap;
+        width: 100%;
+        padding: ($spacer * 0.4) ($spacer * 0.2);
+
+
+        > * {
+          flex: 0 0 auto;
+
+          &:first-child {
+            flex: 0 0 50%;
+          }
+        }
+
+        br {
+          display: none;
+        }
+      }
+
+      strong {
+        text-transform: uppercase;
+      }
+
+      > li:nth-child(even) {
+        background: $gray-100;
+      }
+    }
+  }
+}
+
+/**
+* Reference week
+**/
+.semaineDeRef {
+  @extend .table, .table-boxed;
+
+  tbody {
+    tr:first-child {
+      th {
+        padding: ($spacer * 0.12) ($spacer * 0.66);
+        background: $table-head-bg;
+        border-left: 1px solid gray('200');
+        border-bottom: 3px solid gray('200');
+        text-transform: uppercase;
+        color: $table-head-color;
+      }
+      th:first-child {
+        background: $white;
+      }
+    }
+
+    tr {
+      .time {
+        font-weight: bold;
+      }
+
+      td {
+        padding: ($spacer * 0.15) ($spacer * 0.45)
+      }
+    }
+  }
+}
+
+.local-color {
+  background: theme-color("primary");
+}
+
+.local-color-light {
+  background: darken(theme-color("warning"), 5.2);
+}
+
+.local-color-dark {
+  background: theme-color("info");
+}
+
+.taken {
+  color: white;
+  vertical-align: middle;
+  text-align: center;
+
+  a {
+    color: white !important;
+    text-decoration-color: $white;
+
+    &:hover {
+      font-weight: bold;
+    }
+  }
+}
+
+.coursebook-reference-week > div > div {
+  // these are the items out of the table
+  width: auto !important;
+  margin-right: 0.9rem;
+  font-size: $font-size-sm * 1.07;
+}
+
+/**
+* Legend
+**/
+
+.box.two-cols {
+  display: flex;
+  flex-flow: row wrap;
+
+  .box-col {
+    flex: 0 0 50%;
+  }
+}
+
+.coursebook-legend {
+  h3 {
+    flex: 0 0 100%;
+    font-size: $font-size-base * 1.2;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    font-size: $font-size-sm;
+
+    li {
+      margin-bottom: $spacer * 0.1;
+    }
+  }
+}
+
+/**
+* temporary icons style
```

Comment:
How temporary ? Related to #289 ?

Reply:
Exactly. When we get a final set of icons, we'll integrate them into the coursebook and the study plan!